crux.el

```diff
@@ -606,17 +606,25 @@ as the current user."
   (insert (format-time-string "%c" (current-time))))
 
 ;;;###autoload
-(defun crux-recentf-find-file ()
-  "Find a recent file using `completing-read'."
+(defun crux-recentf-find-file (&optional filter)
```

Comment:
Alternatively you can make this some private helper function and have a single command be used for files and directories depending on whether it was used with a prefix argument (e.g. `C-u C-c f`). I don't have a strong preference either way, though - just something that came to my mind.

Reply:
I don't have a strong preference either. Your call 🙂